lib/view/calendar_page.dart

```diff
@@ -140,10 +144,10 @@ class _MyCalendarState extends State<MyCalendar> with TickerProviderStateMixin {
                     const EdgeInsets.symmetric(horizontal: 8.0, vertical: 4.0),
                 child: ListTile(
                   title: Text(event.toString()),
-                  onTap: () => print('$event tapped!'),
+                  onTap: () => {},
```

Comment:
Best to remove onTap, if tap gesture is not applicable

Reply:
Done.